docs/reference/mapping/fields/source-field.asciidoc

```diff
@@ -7,8 +7,7 @@ searchable), but it is stored so that it can be returned when executing
 _fetch_ requests, like <<docs-get,get>> or <<search-search,search>>.
 
 If disk usage is important to you then have a look at
-<<synthetic-source,synthetic `_source`>> which shrinks disk usage at the cost of
-only supporting a subset of mappings and slower fetches or (not recommended)
+<<synthetic-source,synthetic `_source`>> which shrinks disk usage at the cost of slower fetches or (not recommended)
 <<disable-source-field,disabling the `_source` field>> which also shrinks disk
 usage but disables many features.
```

Comment:
consider defining these briefly up here. if you don't want to define them here, I'd remove the downsides and keep the links only.

```suggestion
- Using <<synthetic-source,synthetic `_source`>>, which reconstructs source content at the time of retrieval instead of storing it on disk. This shrinks disk usage, at the cost of slower access to `_source` in <<docs-get,Get>> and <<search-search,Search>> queries.
- <<disable-source-field,Disabling the `_source` field completely>>. This shrinks disk
usage but disables features that rely on `_source`.
```